audio/src/main/java/edu/artic/audio/AudioDetailsViewModel.kt

```diff
@@ -0,0 +1,80 @@
+package edu.artic.audio
+
+import com.fuzz.rx.bindTo
+import com.fuzz.rx.disposedBy
+import edu.artic.db.models.ArticObject
+import edu.artic.media.audio.AudioPlayerService
+import edu.artic.viewmodel.BaseViewModel
+import io.reactivex.subjects.BehaviorSubject
+import io.reactivex.subjects.Subject
+import javax.inject.Inject
+
+/**
+ * @author Sameer Dhakal (Fuzz)
+ */
+class AudioDetailsViewModel @Inject constructor() : BaseViewModel() {
+    val title: Subject<String> = BehaviorSubject.create()
+    val image: Subject<String> = BehaviorSubject.create()
+    val transcript: Subject<String> = BehaviorSubject.create()
+    val credits: Subject<String> = BehaviorSubject.create()
+    val authorCulturalPlace: Subject<String> = BehaviorSubject.create()
+
+    private val objectObservable: Subject<ArticObject> = BehaviorSubject.create()
+
+    var audioObject: ArticObject? = null
+        set(value) {
+            field = value
+            value?.let {
+                objectObservable.onNext(it)
+            }
+        }
+
+    private var audioService: AudioPlayerService? = null
+
+    fun setService(service: AudioPlayerService?) {
+        audioService = service
+        audioObject = audioService?.articObject
+    }
+
+    override fun onCleared() {
+        super.onCleared()
+        audioService = null
+    }
+
+    init {
+        objectObservable
+                .map {
+                    it.artistCulturePlaceDelim?.replace("\r", "\n") ?: ""
```

Comment:
helper method / extension maybe or something. this might need repeating somewhere.